### PR TITLE
[Storage] Bucket metadata samples

### PR DESCRIPTION
**Pull request by frankyn**
cc: (Go sample) @JesseLovelace

### CODE REVIEW

File: storage/buckets/main.go [hunk 1]
```diff
@@ -137,6 +137,40 @@ func list(client *storage.Client, projectID string) ([]string, error) {
...
+	fmt.Fprintln(w, "RequesterPays: %v", attrs.RequesterPays)
+	fmt.Fprintln(w, "VersioningEnabled: %v", attrs.VersioningEnabled)
+	fmt.Fprintln(w, "LogBucket: %v", attrs.BucketLogging.LogBucket)
+	fmt.Fprintln(w, "LogObjectPrefix: %v", attrs.BucketLogging.LogObjectPrefix)
+	fmt.Fprintln(w, "\n\n\nLabels:")
+	for key, value := range attrs.Labels {
+		fmt.Fprintln(w, "\t%v = %v", key, value)
+	}
+	// [END storage_get_bucket_metadata]
+	return nil
```
Comment (kurtisvg) [inline storage/buckets/main.go]: nit: The other `get*` functions return the object as well - should you do that for this one?

Comment (frankyn) [reply storage/buckets/main.go]: I instead verify the output of the sample in this case. Is that fair?

Comment (kurtisvg) [reply storage/buckets/main.go]: Yeah, it's more consistent with the new format but let consistent with the samples around it to print instead. I'm just pointing it out, I'm don't think it makes a significant different either way

Comment (frankyn) [reply storage/buckets/main.go]: I split it up. PTAL

**Pull request merged**